sklearn/cross_decomposition/_pls.py

```diff
@@ -442,7 +442,8 @@ def predict(self, X, copy=True):
         X -= self._x_mean
         X /= self._x_std
         Ypred = np.dot(X, self.coef_)
-        return Ypred + self._y_mean
+        Ypred += self._y_mean
+        return Ypred.squeeze()
```

Comment:
@glemaitre giving it a second thought, that will break the behavior for the single sample prediction, getting a numpy array with shape `( )` instead of `(1,)`.

Reply:
Passing the parameter `axis` should do the trick only to limit ourselves to the target dimension.

```python
Ypred.squeeze(axis=1)
```

Reply:
You're completely right. Thanks for that.